freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/model/FmsUser.java

```diff
@@ -10,6 +11,8 @@
 
     private String lastName;
 
+    private Optional<String> azureObjectId = Optional.empty();
```

Comment:
see comment on UmsUsersStateProvider for why we can't store this info in this model